extensions/panache/panache-common/runtime/src/main/java/io/quarkus/panache/common/Sort.java

```diff
@@ -40,9 +40,24 @@ public enum Direction {
         Descending;
     }
 
+    /**
+     * Represents the order of null columns.
+     */
+    public enum NullHandling {
```

Comment:
Maybe it would be nice to use the same name as the Hibernate Dialect uses: `NullPrecedence`

Reply:
Good idea